src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -526,6 +539,14 @@ private void AIThink(Random random, in Entity entity, ref MicrobeAI ai, ref Heal
 
                     break;
```

Comment:
This is probably the line that prevents microbes from wanting to use siderophore. I opened a follow up issue: https://github.com/Revolutionary-Games/Thrive/issues/5328